packages/portal-proto/src/features/cohortBuilder/utils.ts

```diff
@@ -6,6 +6,39 @@ import {
 } from "@gff/core";
 import { SummaryFacetInfo } from "./SummaryFacets";
 
+// These new upload facets will not be returned by the mapping API call, hence they are manually defined as upload_facets
+export const upload_facets: Record<string, FacetDefinition> = {
+  "cases.upload.case_id": {
+    description: "Enter/upload cases to filter the current cohort",
+    doc_type: "cases",
+    facet_type: "upload",
+    field: "cases.upload.case_id",
```

Comment:
Why does cases have "upload" in its field name but genes/ssms don't?

Reply:
`cases.case_id` is already returned by the mapping call which is of facet type `exact`. Hence I wanted to give this a different field. I can go ahead and add `upload.` for genes/ssms just to make it consistent. 
Also before sending it as prop to `UploadFacet` I have done this ` field={facet.field.replace("upload.", "")}` which is needed to clear filters correctly.

This has been updated.